src/Castle.Services.Logging.EventLogIntegration/Castle.Services.Logging.EventLogIntegration.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<Import Project="..\..\buildscripts\common.props"></Import>
+
+	<PropertyGroup>
+		<TargetFrameworks>net8.0;net462;netstandard2.0</TargetFrameworks>
+	</PropertyGroup>
+
+	<PropertyGroup>
+		<PackageId>Castle.Core-DiagnosticsLogger</PackageId>
+		<GeneratePackageOnBuild>True</GeneratePackageOnBuild>
+		<PackageOutputPath>../../build/</PackageOutputPath>
+		<AssemblyName>Castle.Services.Logging.EventLogIntegration</AssemblyName>
+		<RootNamespace>Castle.Services.Logging</RootNamespace>
+		<AssemblyTitle>Castle.Services.Logging.EventLogIntegration</AssemblyTitle>
+		<Description>Castle Services DiagnosticsLogger for Windows EventLog</Description>
+		<AssemblyOriginatorKeyFile>..\..\buildscripts\CastleKey.snk</AssemblyOriginatorKeyFile>
+		<SignAssembly>True</SignAssembly>
+		<PublicSign Condition="'$(OS)'=='Unix'">true</PublicSign>
+		<PackageTags>castle logging eventlog</PackageTags>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<Folder Include="Properties\" />
+	</ItemGroup>
```

Comment:
This looks unnecessary, this project does not have a `Properties/` sub-directory.

Reply:
Unnecessary noise removed from csproj-file